docs/proposals/federation.md

```diff
@@ -341,14 +399,23 @@ of our users are in Western Europe, U.S. West Coast" etc).
 
 ## Cross-cluster service discovery
 
-I propose having pods use standard discovery methods used by external clients of Kubernetes applications (i.e. DNS).  DNS might resolve to a public endpoint in the local or a remote cluster. Other than Strictly Coupled applications, software should be largely oblivious of which of the two occurs.
+I propose having pods use standard discovery methods used by external
+clients of Kubernetes applications (i.e. DNS).  DNS might resolve to a
+public endpoint in the local or a remote cluster. Other than Strictly
+Coupled applications, software should be largely oblivious of which of
+the two occurs.
+
 _Aside:_ How do we avoid "tromboning" through an external VIP when DNS
 resolves to a public IP on the local cluster?  Strictly speaking this
-would be an optimization, and probably only matters to high bandwidth,
-low latency communications.  We could potentially eliminate the
-trombone with some kube-proxy magic if necessary. More detail to be
-added here, but feel free to shoot down the basic DNS idea in the mean
-time.
+would be an optimization for some cases, and probably only matters to
+high bandwidth, low latency communications.  We could potentially
```

Comment:
Since you touched, this, you get subjected to one of my grammatical pet peeves.

English is right-associative.

Compound adjectives must be hyphenated, in general:
http://www.grammarbook.com/punctuation/hyphens.asp

high-bandwidth, low-latency